packages/wallet/api/src/wallet.js

```diff
@@ -238,8 +238,8 @@ export function buildRootObject(vatPowers) {
    * @type {WalletBridge}
    */
   const preapprovedBridge = Far('preapprovedBridge', {
-    addOffer(offer) {
-      return walletAdmin.addOffer(offer);
+    addOffer(offer, meta = undefined) {
```

Comment:
jsdoc better for optional.

Reply:
Okay, I added a jsdoc to it.  How do I indicate to the reader of this code (who isn't using Typescript) that the meta argument is optional without overriding the underlying default?